core/src/main/java/com/github/kaktushose/jda/commands/message/emoji/Component.java

```diff
@@ -0,0 +1,6 @@
+package com.github.kaktushose.jda.commands.message.emoji;
+
+sealed interface Component permits Component.EmojiReference, Component.Literal {
```

Comment:
can we come up with a better name? this might be confused with message components

Reply:
It's not part of the user api, just a package private container class